proc/proc.go

```diff
@@ -699,3 +725,50 @@ func (dbp *Process) getGoInformation() (ver GoVersion, isextld bool, err error)
 	}
 	return
 }
+
+func (dbp *Process) ConvertEvalScope(gid, frame int) (*EvalScope, error) {
+	var g *G
+
+	if gid == -1 {
+		g = dbp.SelectedGoroutine
+		if g == nil {
+			return dbp.CurrentThread.Scope()
+		}
+	} else {
+		gs, err := dbp.GoroutinesInfo()
```

Comment:
Probably worth extracting this into a method off of `Process`, something like `FindGoroutine`.